packages/ui/src/components/Search/FLAutoComplete.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { useCallback, useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import { SearchOutlined } from '@ant-design/icons';
-import { AutoComplete, Input, Select, Typography } from 'antd';
+import { AutoComplete, Select } from 'antd';
 import { debounce } from 'lodash';
```

Comment:
dont import lodash like that, It blows the bundle size. Always use absolution path.